src/quo2/components/channel_avatar.cljs

```diff
@@ -0,0 +1,47 @@
+(ns quo2.components.channel-avatar
+  (:require [quo.design-system.colors :as colors]
+            [quo2.foundations.colors :as f-colors]
+            [quo.react-native :as rn]
+            [clojure.string :as clj-string]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(defn channel-avatar [{:keys [big? dark? lock-status icon x y]}]
+  [rn/view {:style {:width (if big? 32 24)
+                    :height (if big? 32 24)
+                    :top (if (or (clj-string/blank? x)
+                                 (js/isNaN x))
+                           0
+                           (js/parseInt x))
+                    :left (if (or (clj-string/blank? y)
+                                  (js/isNaN y))
+                            0
+                            (js/parseInt y)) 
+                    :border-radius 100
```

Comment:
why border-radius 100?

Reply:
Changed it to match the width and height and it looks circular and cool 😄 
I tried a high value and it was circular so I left it that way 😅

Reply:
Done